client/http-client/src/transport.rs

```diff
@@ -74,16 +74,15 @@ impl HttpTransportClient {
 						.with_native_roots()
 						.https_or_http()
 						.enable_http1()
-						.wrap_connector(connector),
+						.build(),
 					CertificateStore::WebPki => hyper_rustls::HttpsConnectorBuilder::new()
 						.with_webpki_roots()
 						.https_or_http()
 						.enable_http1()
-						.wrap_connector(connector),
+						.build(),
 					_ => return Err(Error::InvalidCertficateStore),
 				};
-				let client = Client::builder().build::<_, hyper::Body>(connector);
-				HyperClient::Https(client)
+				HyperClient::Https(Client::builder().build::<_, hyper::Body>(connector))
```

Comment:
I'm not really sure what the impact of this change is; was wrapping the HttpConnector causing an issue?

Reply:
Ah I see; I just read https://github.com/rustls/hyper-rustls/issues/169. Interesting!

My reading/assumption was that the wrapping would work as we had expected!